src/num/init.c

```diff
@@ -13,7 +13,9 @@
 #define _GNU_SOURCE
 #include <stdlib.h>
 #include <fenv.h>
+#if 0
```

Comment:
sys/resource.h is a POSIX-specific header, but the function that would use it is also commented out, so it didn't cause a problem to comment it temporarily out.